sources/packages/web/src/components/layouts/aest/AESTApplicationSideBar.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <v-navigation-drawer app class="body-background">
+    <v-list dense nav>
+      <v-list-item @click="studentMenu.command">
+        <v-list-item-icon>
+          <font-awesome-icon :icon="studentMenu.icon" class="mr-2" />
+        </v-list-item-icon>
+        <v-list-item-content>
+          <v-list-item-title>{{ studentMenu.label }} </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+      <v-list-item
+        v-for="ppAppliation in relatedPPApplications"
+        :key="ppAppliation.label"
+        @click="ppAppliation.command"
+      >
+        <v-list-item-icon>
+          <font-awesome-icon :icon="ppAppliation.icon" class="mr-2" />
+        </v-list-item-icon>
+        <v-list-item-content>
+          <v-list-item-title>{{ ppAppliation.label }}</v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+    </v-list>
+  </v-navigation-drawer>
+</template>
+
+<script lang="ts">
+import { useRouter } from "vue-router";
+import { ref, onMounted } from "vue";
+import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
+import { MenuModel, SupportingUserType } from "@/types";
+import { SupportingUsersService } from "@/services/SupportingUserService";
+
+export default {
+  props: {
+    studentId: {
+      type: Number,
+      required: false,
```

Comment:
Is this not required?

Reply:
initially, I was planning to create separate parent  routers for student application and supporting users, so this was actually not required, but in my next PR, supporting users are children of student application, so, it is `true` and the change will be in my next PR
![image](https://user-images.githubusercontent.com/77353155/155432623-f834c6fa-2e9f-4be2-9a81-fcb401a9133f.png)

Reply:
Thanks for the clarification.